Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -677,6 +684,28 @@ bool EventsRefactorer::ReplaceStringInConditions(
   return somethingModified;
 }
 
+bool EventsRefactorer::ReplaceStringInStringEvent(gd::ObjectsContainer& project,
+                                                  gd::ObjectsContainer& layout,
+                                                  gd::BaseEvent& event,
+                                                  gd::String toReplace,
+                                                  gd::String newString,
+                                                  bool matchCase) {
+  vector<gd::String> newStringEvents;

```

Comment:
Nitpicking:
```suggestion
  vector<gd::String> newEventStrings;
```

There is a single event, but with (potentially) multiple strings.